charts/tidb-operator/templates/controller-manager-deployment.yaml

```diff
@@ -50,3 +50,12 @@ spec:
                 fieldPath: metadata.namespace
           - name: TZ
             value: {{ .Values.timezone | default "UTC" }}
+        volumeMounts:
+        - mountPath: /var/lib/tls
+          name: tls
+          readOnly: true
+      volumes:
+      - name: tls
+        secret:
+          defaultMode: 420
+          secretName: client-tls
```

Comment:
what if more than one TiDB clusters are managed by the operator with different client certs?